foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/OpenAiChatCompletionRequest.java

```diff
@@ -126,7 +128,9 @@ public class OpenAiChatCompletionRequest {
   @Nullable List<ChatCompletionTool> tools;
 
   /** Option to control which tool is invoked by the model. */
-  @Nullable ChatCompletionToolChoiceOption toolChoice;
+  @With(AccessLevel.PRIVATE)
```

Comment:
This is a breaking change, It's fine because we didn't release but be careful with those lombok annotations, they can create too much public API